odfuzz/fuzzer.py

```diff
@@ -102,6 +104,9 @@ def __init__(self, dispatcher, entities, database, output_handler, asynchronous)
             self._queryable_factory = SingleQueryable
             self._dispatch = self._get_single_response
 
+        if Config.fuzzer.use_encoder:
+            self._decode_queries = lambda x: x
```

Comment:
Not runinng localy so just just using "Eyeball Mark I" - should this logic be exactly opposite?

I am reading this as "if use_encoder is true ( and that is the deafult value), then use no decoder". I am somehow expecting here the usage of utils.decode_string().

also rename the value pls - "_decode_queries" indicates more boolean value and use in IF statement, similar to _asynchronous or _first_touch. You would not expect such type in _test_num (iditicates int) or _queryable_factory (indicates something.

Reply:
Oopsie, you are right. This was done unwittingly. I will try to rename the method.